src/Controls/src/Core/HandlerImpl/Window.Android.cs

```diff
@@ -0,0 +1,18 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Android.App;
+using AndroidX.AppCompat.App;
+
+namespace Microsoft.Maui.Controls
+{
+	public partial class Window
+	{
+		internal AppCompatActivity NativeActivity
```

Comment:
Should we just use Activity for this? Just in case Google deprecates AppCompat :)

Reply:
I needed it to be AppCompatActivity so I could get access to the `OnBackPressedDispatcher` plus I just set it as internal so should be fine for now